templates/shared/forms/interactive/robotics_ros.html

```diff
@@ -61,35 +55,39 @@ <h3 class="p-heading--five">How many devices will require ESM coverage yearly?</
     <div class="js-pagination js-pagination--2 u-hide">
 
       <div class="js-formfield">
-        <h3 class="p-heading--five">How often will you update your robot yearly?</h3>
+        <h3 class="p-heading--five">How often will you update your robot?</h3>
         <div class="row u-no-padding">
           <div class="col-4 u-sv3">
-            <input type="radio" id="update-frequency-every-12-months" name="update-frequency" value="Every 12 months">
-            <label for="update-frequency-every-12-months">Every 12 months</label>
-            <input type="radio" id="update-frequency-every-6-months" name="update-frequency" value="Every 6 months">
-            <label for="update-frequency-every-6-months">Every 6 months</label>
-            <input type="radio" id="update-frequency-every-3-months" name="update-frequency" value="Every 3 months">
-            <label for="update-frequency-every-3-months">Every 3 months</label>
+            <input type="radio" id="update-frequency-never" name="update-frequency" value="Never">
+            <label for="update-frequency-never">Never</label>
+            <input type="radio" id="update-frequency-in-real-time" name="update-frequency" value="In real time">
+            <label for="update-frequency-in-real-time">In real time</label>
+            <input type="radio" id="update-frequency-every-6-hours" name="update-frequency" value="Every 6 hours">
+            <label for="update-frequency-every-6-hours">Every 6 hours</label>
+            <input type="radio" id="update-frequency-every-day" name="update-frequency" value="Every day">
+            <label for="update-frequency-every-day">Every day</label>
           </div>
```

Comment:
We have a blank option

Reply:
![form](https://user-images.githubusercontent.com/78618068/109675481-c0bb0380-7b6f-11eb-9694-1a1fc18936ac.png)